st/clitests/backgrounddelete_spec.py

```diff
@@ -140,8 +140,8 @@ def perform_head_object(oid_dict):
 load_and_update_config(account_response_elements['AccessKeyId'], account_response_elements['SecretKey'])
 
 #Clear probable delete list index
-s3kvs.clean_all_data()
-s3kvs.create_s3root_index()
+#s3kvs.clean_all_data()
+#s3kvs.create_s3root_index()
```

Comment:
why these commented ?

Reply:
These lines recreate global indices after s3server creates them and saves their pool version id & layout type.